ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestContext.scala

```diff
@@ -63,4 +65,9 @@ final class LedgerTestContext(
   def semanticTesterLedger(parties: Set[Ref.Party], packages: Map[Ref.PackageId, Ast.Package]) =
     new SemanticTesterLedger(bindings)(parties, packages)(this)
 
+  def extractErrorStatus[A](future: Future[A]): Future[Status.Code] =
```

Comment:
Very nice method. :slightly_smiling_face. This does not need to be in the `LedgerTestContext`, but rather needs an `ExecutionContext` for it to run. I would suggest moving it in the `LedgerTestSuite` taking an implicit `ExecutionContext` (of which `LedgerTestContext` is an instance).

Reply:
Given how it's used, a good name could be something like `expectGrpcErrorCode`.

Reply:
Addressed by acb912d.